tests/lib/cart-item-test.js

```diff
@@ -14,11 +14,6 @@ describe('CartItem', () => {
       expect(item.is_valid_item()).toBe(false);
     });
 
-    it('rejects cart item with invalid price', () => {
```

Comment:
I guess this price validation was removed almost 5 years ago! `on Oct 23, 2020` via https://github.com/creditkey/creditkey-js/pull/60/files